### PR TITLE
fix: update docs with new cli commands

### PR DESCRIPTION
**Pull request by arihantbansal**
Pls review @ShrinathNR

### CODE REVIEW

File: docs/src/tutorial.md [hunk 1]
```diff
@@ -73,24 +73,16 @@ Remember what Poseidon does for you? Here’s a quick recap:
...
-$ anchor init vote_program
-$ cd vote_program
-
-# Setup TypeScript program inside the repository anchor created
-$ mkdir -p ts-programs/src
-$ cd ts-programs
-$ yarn init -y
-$ yarn add @solanaturbine/poseidon
-$ touch src/voteProgram.ts
+$ poseidon init vote_program
```
Comment (ShrinathNR) [inline docs/src/tutorial.md]: The init command does not accepts '_'. 
It throws, _Invalid project name. Project names must start with a letter and contain only alphanumeric characters and hyphens._

Comment (arihantbansal) [reply docs/src/tutorial.md]: Ah yes, should be `poseidon init vote-program`

**Comment by danielAsaboro** [comment]
this pr breaks the vote tutorial completely, lots of fixes needed there;

**Comment by ShrinathNR** [comment]
gm, you need to `anchor build` after tranpilation. I have added that in the tutorial docs now.

**Comment by danielAsaboro** [comment]
this was the [issue](https://stackoverflow.com/questions/78036808/error-when-running-anchor-test-rust-directory-not-found), fixed...thank you

**Pull request merged**